config/default.yml

```diff
@@ -1505,6 +1505,11 @@ Lint/PercentSymbolArray:
   Enabled: true
   VersionAdded: '0.41'
 
+Lint/RaiseException:
+  Description: Checks for `raise` or `fail` statements which is raise `Exception` class.
+  Enabled: true
+  VersionAdded: '0.75'
```

Comment:
Is this in the style guide? If so - let's add a link to the rule.

Reply:
no, we have only this one https://github.com/rubocop-hq/ruby-style-guide#no-blind-rescues

Reply:
Guess we can add something on the subject there then.

Reply:
@bbatsov Did you mean  in style guide?

Reply:
Yep.

Reply:
@bbatsov  https://github.com/rubocop-hq/ruby-style-guide/pull/775